x_dummy: 0.1
y_dummy: -0.35
z_dummy: 0.3